docs/sphinx/dev_guide/ci.rst

```diff
@@ -8,13 +8,76 @@
 
 .. _ci:
 
-**********************
-Continuous Integration
-**********************
+************************************
+Continuous Integration (CI) Testing
+************************************
+
+The RAJA project employes multiple tools to run its tests for each GitHub
+*pull request*, all of which must pass before the pull request can be merged.
+These tools include:
+
+  * **Travis CI.** This runs builds for a Linux system for multiple versions of
+    the GNU and clang compilers, and one version of the Intel, nvcc (CUDA), and 
+    HIP compilers. RAJA tests are run each non-GPU build.
+  * **Appveyor.** This runs builds and tests for a Windows environment for two
+    versions of the Visual Studio compiler.
+  * **Gitlab CI.** Runs builds and tests on platforms in the Livermore
+    Computing *Collaboration Zone*. This is a recent addition for RAJA and
+    is a work-in-progress to get full coverage of compilers and tests we
+    need to exercise.
+
+These tools integrate seamlessly with GitHub. So they will automatically
```

Comment:
If I am to be picky.... :D
Either "So, they will..." or just "They will..."
(typo)

Reply:
Not "Ergo, they will...." :-)  Changed.